script_test.go

```diff
@@ -2276,6 +2373,22 @@ func ExamplePipe_Get() {
 	// You said: hello
 }
 
+func ExamplePipe_Hash() {
+	sum, err := script.Echo("hello world").Hash(sha512.New())
+	if err != nil {
+		panic(err)
+	}
+	fmt.Println(sum)
+	// Output:
+	// 309ecc489c12d6eb4cc40f50c902f2b4d0ed77ee511a7c7a9bcd3ca86d4cd86f989dd35bc5ff499670da34255b45b0cfd830e81f605dcf7dc5542e93ae9cd76f
+}
+
+func ExamplePipe_HashSums() {
+	script.Echo("testdata/test.txt").HashSums(sha512.New()).Stdout()
```

Comment:
Would it be clearer if we used two files here, to show that the method produces one hash per file?

Reply:
done!